todo_app/data/session_items.py

```diff
@@ -63,35 +55,34 @@ def add_item(title):
         item: The saved item.
     """
 
-    url = f"https://api.trello.com/1/cards"
-    data = {"idList": os.getenv("LIST_ID_TO_DO"), "name": title}
-    response = requests.post(url, data=data, params=query)
+    data = {
+        "status": "Not Started",
+        "name": title,
+        "dateLastActivity": datetime.now(),
+        "id": random.randint(0, 1000),
+    }
+    collection = get_client.todo_app.items
```

Comment:
You're not calling the get_client function here (maybe swapped from a variable at some point) which leads to errors - note the same problem in a few lines below!
```suggestion
    collection = get_client().todo_app.items
```